karmagrambot/commands.py

```diff
@@ -36,7 +36,27 @@ def karma(bot: Bot, update: Update):
 
     update.message.reply_text(karma)
 
+def karmas(bot: Bot, update: Update):
+    """Shows the top 10 karmas in a given group, if the groups doesn't have at least 10 users, show the maximum amount
+
+    Args:
+        bot: The object that represents the Telegram Bot.
+        update: The object that represents an incoming update for the bot to handle.
+
+    Returns:
+        Doesn't return anything, but answer the user with the top 10 karmas in the group.
+    """
+
+    top_users = analytics.get_top_10_karmas(update.message.chat.id)
+
+    response = ''
```

Comment:
These three lines can be changed to

`response = ''.join(f'{i} - {user.name} ({user.karma})' for i, user in enumerate(top_user, 1))`